resources/netcanv-matchmaker.desktop

```diff
@@ -2,7 +2,7 @@
 Type=Application
 Encoding=UTF-8
 Name=NetCanv Matchmaker
-Comment=Server for Multiplayer Paint
+Comment=Matchmaker for NetCanv
```

Comment:
I say, we should use the same descriptions/comments on both Windows and Linux. The Windows description for the matchmaker is as follows:
https://github.com/liquidev/netcanv/blob/ec91593adaabb1956fb12a94af30b895f9600b4a/build/windows-icons.ps1#L60